index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+      </style>
+    <title>Document</title>
+    
+</head>
+<body>
+    <header class="navegation-menu">
+    <div class="humburguer"><i href="#" class="fa fa-bars"></i>
```

Comment:
- [ ] It would be better if you use a `nav` tag instead of the `div` tag to wrap your hamburger menu. This is for several reasons including semantics and user experience.

Reply:
@Graciano1997 , Yes, it is a good practice to use the <nav> tag inside the <header> tag in HTML. The <nav> tag is used to define a section of a web page that contains navigation links, such as menus, site maps, or other navigation-related content. The <header> tag, on the other hand, is used to define the introductory content or a container for a set of navigational elements.